library/foundations/scripts/save-assets.ts

```diff
@@ -85,11 +98,13 @@ loadLayersFile(snapshotFileName)
   .then((assets) =>
     // for every asset, download it and convert it to webp
     Promise.all(
-      assets.map(({ value, ...rest }) =>
-        findAssetSizeAndDimensions(value).then((metadata) => ({
-          ...rest,
-          ...metadata,
-        }))
+      assets.map(({ value, ...rest }, i) =>
+        batchDelay(i)
```

Comment:
batch asset requests to avoid upstream (likely rate-limiting) timeouts.
<img width="808" alt="Screenshot 2023-07-19 at 10 26 58 AM" src="https://github.com/rangle/radius-monorepo-react/assets/1680390/ce01c5e5-8bff-4bdb-a1ee-2a52e88d3b7c">